contrib/Snaphu/SConscript

```diff
@@ -16,8 +16,11 @@ envSnaphu = envcontrib.Clone()
 project = 'Snaphu'
 package = envSnaphu['PACKAGE']
 envSnaphu['PROJECT'] = project
+if "-O2" in envSnaphu['CCFLAGS']: envSnaphu['CCFLAGS'].remove('-O2')
+envSnaphu.Append(CCFLAGS=['-O3'])
```

Comment:
I make this change refers to the original Makefile of snaphu. Because the unwrapping often takes lots of time, I make this change to see if it could save some time. I think it's okay to change the flag back to "-O2".